08/28/20 18:40:16 - INFO     - _initialise: 

# ============================================== #
#               THE GIST PIPELINE                #
#                 Version 3.0.0                  #
# ============================================== #

08/28/20 18:40:16 - INFO     - _readData: Using the read-in routine for SOAR
08/28/20 18:40:16 - INFO     - SOAR: Reading the SOAR long-slit spectrum: ./inputData/binned3drs0010crop.fits
08/28/20 18:40:17 - INFO     - SOAR: No error extension found. Estimating the error spectra with the der_snr algorithm
08/28/20 18:40:17 - INFO     - SOAR: Extracting spatial information:
                                     * Spatial coordinates are centred to [0.0, 58.0]
                                     * Spatial pixelsize is 0.8699999999999999
08/28/20 18:40:17 - INFO     - SOAR: Extracting spectral information:
                                     * Shortened spectra to wavelength range from 4400 to 6900 Angst.
                                     * Spectral pixelsize in velocity space is 103.14164807117601 km/s
08/28/20 18:40:17 - INFO     - SOAR: Removing all spaxels containing nan or having a negative median flux:
                                     * Of 46 in the cube, 42 are accepted and 4 removed
08/28/20 18:40:17 - INFO     - SOAR: Computing the signal-to-noise ratio in the wavelength range from 6900A to 6900A.
08/28/20 18:40:17 - INFO     - SOAR: Finished reading the SOAR long-slit spectrum!
08/28/20 18:40:17 - INFO     - _spatialMasking: Using the spatial masking routine for default
08/28/20 18:40:17 - INFO     - default: Masking defunct spaxels: 0 spaxels are rejected.
08/28/20 18:40:17 - INFO     - default: Masking low signal-to-noise spaxels: 14 spaxels are rejected.
08/28/20 18:40:17 - INFO     - default: No maskfile specified.
08/28/20 18:40:17 - INFO     - default: Combined mask: 14 spaxels are rejected.
08/28/20 18:40:17 - INFO     - default: Wrote mask file: ./results/./rs0010_snr8_2/rs0010_snr8_2_mask.fits
08/28/20 18:40:17 - INFO     - _spatialBinning: Using the spatial binning routine for voronoi
08/28/20 18:40:17 - INFO     - voronoi: Defining the Voronoi bins
08/28/20 18:40:17 - INFO     - voronoi: 3 Voronoi bins generated!
08/28/20 18:40:17 - INFO     - voronoi: Wrote Voronoi table: ./results/./rs0010_snr8_2/rs0010_snr8_2_table.fits
08/28/20 18:40:17 - INFO     - _prepareSpectra: Using the routine for default
08/28/20 18:40:17 - INFO     - default: Applied spatial bins to lin-data
08/28/20 18:40:17 - INFO     - default: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_BinSpectra_linear.fits
08/28/20 18:40:17 - INFO     - default: Log-rebinned the spectra
08/28/20 18:40:17 - INFO     - default: Log-rebinned the error spectra
08/28/20 18:40:17 - INFO     - default: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_AllSpectra.fits
08/28/20 18:40:17 - INFO     - default: Applied spatial bins to log-data
08/28/20 18:40:17 - INFO     - default: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_BinSpectra.fits
08/28/20 18:40:17 - INFO     - _stellarKinematics: Using the stellarKinematics routine 'ppxf.py'
08/28/20 18:40:17 - INFO     - ppxf: Using full spectral library for PPXF
08/28/20 18:40:17 - INFO     - _prepareTemplates: Using the routine for 'miles.py'
08/28/20 18:40:19 - INFO     - miles: Prepared the stellar population templates
08/28/20 18:40:19 - INFO     - ppxf: Using V and SIGMA from the MasterConfig file as initial guesses
08/28/20 18:40:19 - INFO     - ppxf: Running PPXF in parallel mode
08/28/20 18:40:24 - INFO     - ppxf: Running PPXF on 3 spectra took 4.51s using 16 cores
08/28/20 18:40:24 - INFO     - ppxf: There were no problems in the analysis.
08/28/20 18:40:24 - INFO     - ppxf: Calculating Lambda_R
08/28/20 18:40:24 - INFO     - ppxf: To obtain correct lambda_R measurements, the coordinate system must be centred on the centre of the galaxy. Use the 'READ_DATA|ORIGIN' parameter to do so.
08/28/20 18:40:24 - INFO     - ppxf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_kin.fits
08/28/20 18:40:24 - INFO     - ppxf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_kin-bestfit.fits
08/28/20 18:40:24 - INFO     - ppxf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_kin-optimalTemplates.fits
08/28/20 18:40:24 - ERROR    - _plotting: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
Traceback (most recent call last):
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/_plotting.py", line 18, in generatePlots
    gistPlot_kin.plotMaps('KIN', config['GENERAL']['OUTPUT'])
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/gistPlot_kin.py", line 163, in plotMaps
    XY_Triangulation = Triangulation(X-pixelsize/2, Y-pixelsize/2)                      # Create a mesh from a Delaunay triangulation
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/matplotlib/tri/triangulation.py", line 54, in __init__
    self.triangles, self._neighbors = _qhull.delaunay(x, y)
RuntimeError: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
08/28/20 18:40:24 - ERROR    - _plotting: Failed to produce stellar kinematics maps.
08/28/20 18:40:24 - INFO     - _emissionLines: Using the emissionLines routine 'gandalf.py'
08/28/20 18:40:24 - WARNING  - gandalf: It is currently not possible to derive errors with pyGandALF in a Python3 environment. An updated version of pyGandALF will be released soon.
08/28/20 18:40:24 - WARNING  - gandalf: The emission-line analysis continues without an error estimation.
08/28/20 18:40:24 - INFO     - gandalf: Using full spectral library for GANDALF on BIN level
08/28/20 18:40:24 - INFO     - _prepareTemplates: Using the routine for 'miles.py'
08/28/20 18:40:27 - INFO     - miles: Prepared the stellar population templates
08/28/20 18:40:27 - INFO     - gandalf: Running GANDALF in parallel mode
08/28/20 18:40:45 - INFO     - gandalf: Running GANDALF on 3 spectra took 17.97s using 16 cores
08/28/20 18:40:45 - INFO     - gandalf: There were no problems in the analysis.
08/28/20 18:40:45 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas_BIN.fits
08/28/20 18:40:45 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-bestfit_BIN.fits
08/28/20 18:40:45 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-emission_BIN.fits
08/28/20 18:40:45 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-cleaned_BIN.fits
08/28/20 18:40:45 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-weights_BIN.fits
08/28/20 18:40:45 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-optimalTemplate_BIN.fits
08/28/20 18:40:45 - INFO     - gandalf: Using previously extracted optimal templates from the GANDALF BIN level on SPAXEL level
08/28/20 18:40:45 - INFO     - gandalf: Running GANDALF in parallel mode
08/28/20 18:40:51 - INFO     - gandalf: Running GANDALF on 42 spectra took 6.07s using 16 cores
08/28/20 18:40:51 - INFO     - gandalf: There were no problems in the analysis.
08/28/20 18:40:51 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas_SPAXEL.fits
08/28/20 18:40:51 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-bestfit_SPAXEL.fits
08/28/20 18:40:51 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-emission_SPAXEL.fits
08/28/20 18:40:51 - INFO     - gandalf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-cleaned_SPAXEL.fits
08/28/20 18:40:51 - ERROR    - _plotting: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
Traceback (most recent call last):
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/_plotting.py", line 33, in generatePlots
    gistPlot_gas.plotMaps(config['GENERAL']['OUTPUT'], 'BIN', True)
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/gistPlot_gas.py", line 270, in plotMaps
    plot_line(data, LEVEL, FROM_PIPELINE, INTERACTIVE, SAVE_AFTER_INTERACTIVE, line, vminmax, 0.20, X, Y, FLUX, pixelsize, outdir, rootname)
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/gistPlot_gas.py", line 124, in plot_line
    XY_Triangulation = Triangulation(X-pixelsize/2, Y-pixelsize/2)                      # Create a mesh from a Delaunay triangulation
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/matplotlib/tri/triangulation.py", line 54, in __init__
    self.triangles, self._neighbors = _qhull.delaunay(x, y)
RuntimeError: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
08/28/20 18:40:51 - ERROR    - _plotting: Failed to produce maps from the emission-line analysis.
08/28/20 18:40:51 - INFO     - _starFormationHistories: Using the starFormationHistories routine 'ppxf.py'
08/28/20 18:40:51 - INFO     - _prepareTemplates: Using the routine for 'miles.py'
08/28/20 18:40:54 - INFO     - miles: Prepared the stellar population templates
08/28/20 18:40:54 - INFO     - ppxf: Using emission-subtracted spectra at ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-cleaned_BIN.fits
08/28/20 18:40:54 - INFO     - ppxf: Stellar kinematics are FIXED to the results obtained before.
08/28/20 18:40:54 - INFO     - ppxf: Running PPXF in parallel mode
08/28/20 18:41:28 - INFO     - ppxf: Running PPXF on 3 spectra took 34.27s using 16 cores
08/28/20 18:41:28 - INFO     - ppxf: There were no problems in the analysis.
08/28/20 18:41:28 - INFO     - ppxf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_sfh.fits
08/28/20 18:41:28 - INFO     - ppxf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_sfh-weights.fits
08/28/20 18:41:28 - INFO     - ppxf: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_sfh-bestfit.fits
08/28/20 18:41:28 - ERROR    - _plotting: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
Traceback (most recent call last):
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/_plotting.py", line 48, in generatePlots
    gistPlot_sfh.plotMaps('SFH', config['GENERAL']['OUTPUT'])
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/gistPlot_sfh.py", line 176, in plotMaps
    XY_Triangulation = Triangulation(X-pixelsize/2, Y-pixelsize/2)                      # Create a mesh from a Delaunay triangulation
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/matplotlib/tri/triangulation.py", line 54, in __init__
    self.triangles, self._neighbors = _qhull.delaunay(x, y)
RuntimeError: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
08/28/20 18:41:28 - ERROR    - _plotting: Failed to produce SFH maps.
08/28/20 18:41:28 - INFO     - _lineStrengths: Using the lineStrengths routine 'default.py'
08/28/20 18:41:28 - INFO     - default: Using emission-subtracted spectra at ./results/./rs0010_snr8_2/rs0010_snr8_2_gas-cleaned_BIN.fits
08/28/20 18:41:28 - INFO     - default: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_ls-cleaned_linear.fits
08/28/20 18:41:28 - INFO     - default: Running lineStrengths in parallel mode
08/28/20 18:41:29 - INFO     - default: Running lineStrengths on 3 spectra took 0.60s using 16 cores
08/28/20 18:41:29 - INFO     - default: There were no problems in the analysis.
08/28/20 18:41:29 - INFO     - default: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_ls_OrigRes.fits
08/28/20 18:41:29 - INFO     - default: Reading ./results/./rs0010_snr8_2/rs0010_snr8_2_ls-cleaned_linear.fits
08/28/20 18:41:29 - INFO     - default: Loading LS model file at ./spectralTemplates/MILES_KB_LIS8.4.fits
08/28/20 18:41:29 - INFO     - default: Running lineStrengths in parallel mode
08/28/20 18:41:30 - INFO     - default: Running lineStrengths on 3 spectra took 0.83s using 16 cores
08/28/20 18:41:30 - INFO     - default: There were no problems in the analysis.
08/28/20 18:41:30 - INFO     - default: Wrote: ./results/./rs0010_snr8_2/rs0010_snr8_2_ls_AdapRes.fits
08/28/20 18:41:30 - ERROR    - _plotting: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
Traceback (most recent call last):
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/_plotting.py", line 61, in generatePlots
    gistPlot_ls.plotMaps(config['GENERAL']['OUTPUT'], 'ORIGINAL')
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/plotting/gistPlot_ls.py", line 166, in plotMaps
    XY_Triangulation = Triangulation(X-pixelsize/2, Y-pixelsize/2)                      # Create a mesh from a Delaunay triangulation
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/matplotlib/tri/triangulation.py", line 54, in __init__
    self.triangles, self._neighbors = _qhull.delaunay(x, y)
RuntimeError: Error in qhull Delaunay triangulation calculation: input inconsistency (exitcode=1); use python verbose option (-v) to see original qhull error.
08/28/20 18:41:30 - ERROR    - _plotting: Failed to produce line strength maps.
08/28/20 18:41:38 - INFO     - MainPipeline: The GIST completed successfully.
08/28/20 18:41:38 - ERROR    - _initialise: Uncaught Exception
Traceback (most recent call last):
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/bin/gistPipeline", line 11, in <module>
    load_entry_point('gistPipeline==3.0.0', 'console_scripts', 'gistPipeline')()
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/MainPipeline.py", line 206, in main
    runGIST(dirPath, galindex)
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/MainPipeline.py", line 76, in runGIST
    config = _initialise.addPathsToConfig(config, dirPath)
  File "/afs/cas.unc.edu/users/m/u/mugpol/anaconda3/envs/gistenv/lib/python3.6/site-packages/gistPipeline-3.0.0-py3.6.egg/gistPipeline/initialise/_initialise.py", line 130, in addPathsToConfig
    config['GENERAL']['INPUT'] = os.path.join(line[1], config['GENERAL']['INPUT'])
TypeError: 'NoneType' object is not subscriptable
